CodeBeam.MudBlazor.Extensions/Components/Stepper/MudStepTemplate.razor

```diff
@@ -0,0 +1,46 @@
+﻿@namespace MudExtensions
+@inherits ComponentBase
+
+@if (Step != null)
+{
+    @if (Step.Status == MudExtensions.Enums.StepStatus.Completed)
+    {
+        <MudBadge Content="3" Overlap="true" Class="" Icon="@Icons.Material.Filled.Done">
```

Comment:
Is there a reason to add `Content="3" here?

Reply:
no reason, i must have missed removing that.